pkl-cli/src/test/kotlin/org/pkl/cli/CliProjectPackagerTest.kt

```diff
@@ -858,17 +867,18 @@ class CliProjectPackagerTest {
         consoleWriter = out
       )
       .run()
+    val sep = File.separatorChar
     assertThat(out.toString())
-      .isEqualTo(
+      .isEqualToNormalizingNewlines(
         """
-      .out/project1@1.0.0/project1@1.0.0.zip
-      .out/project1@1.0.0/project1@1.0.0.zip.sha256
-      .out/project1@1.0.0/project1@1.0.0
-      .out/project1@1.0.0/project1@1.0.0.sha256
-      .out/project2@2.0.0/project2@2.0.0.zip
-      .out/project2@2.0.0/project2@2.0.0.zip.sha256
-      .out/project2@2.0.0/project2@2.0.0
-      .out/project2@2.0.0/project2@2.0.0.sha256
+      .out${sep}project1@1.0.0${sep}project1@1.0.0.zip
+      .out${sep}project1@1.0.0${sep}project1@1.0.0.zip.sha256
+      .out${sep}project1@1.0.0${sep}project1@1.0.0
+      .out${sep}project1@1.0.0${sep}project1@1.0.0.sha256
+      .out${sep}project2@2.0.0${sep}project2@2.0.0.zip
+      .out${sep}project2@2.0.0${sep}project2@2.0.0.zip.sha256
+      .out${sep}project2@2.0.0${sep}project2@2.0.0
+      .out${sep}project2@2.0.0${sep}project2@2.0.0.sha256
```

Comment:
Log lines on Windows will output CRLF endings, and also output `\` as the path separator.

The rationale is that these are log lines, which are meant to be useful for the host machine specifically.

Reply:
Can you elaborate on what is not consistent here?